frontend/src/components/Comment/PostComment.js

```diff
@@ -1,15 +1,23 @@
-import React from "react";
+import React, { useState }  from 'react'
 import "./CommentAPI";
 import { postComment } from "./CommentAPI";
-import "./PostComment.scss";
+import "./SCSS/PostComment.scss";
 
-class PostComment extends React.Component {
+export const PostComment = (props) => {
 
-    comment(e) {
+    const [textAreaText, settextAreaText] = useState("");
+
+    const handleTyping = (e) => {
+        settextAreaText(e.target.value);
+    }
+
+    //Post a comment
+    const comment = (e) => {
         e.preventDefault();
-        postComment(e.target.comment.value, this.props.postId).then(
+        postComment(e.target.comment.value, props.postId).then(
             () => {
-                this.props.onCommentPosted();
+                props.onCommentPosted();
+                settextAreaText("");
             }
         ).catch((e) => {
             // todo: proper error feedback
```

Comment:
Is this still something that has to be done?

Reply:
This is such that when you submit, the textarea clear out

Reply:
i think @emilie-martin is referring to the `todo`